Source/macOS/OIDRedirectHTTPHandler.m

```diff
@@ -76,10 +76,18 @@ - (NSURL *)startHTTPListener:(NSError **)returnError {
       *returnError = error;
     }
     return nil;
-  } else {
+  } else if ([_httpServ hasIPv4Socket]) {
```

Comment:
This was intentional, as I didn't want to change the behavior of the library dramatically without any discussion. Handling a potential IPv6-only case is easy enough to do, I'll add it even though it's unlikely to be hit any time soon.